camayoc/tests/qcs/test_credentials.py

```diff
@@ -223,4 +333,20 @@ def test_delete(cleanup):
         the deleted credential.
     :caseautomation: notautomated
     """
-    pass
+    host_credentials = []
+    for _ in range(random.randint(2, 5)):
+        cred = HostCredential(client=shared_client, password=uuid4())
+        cred.create()
+        # add the credential to the list to destroy after the test is done
+        cleanup.append(cred)
+        assert_matches_server(cred)
+        host_credentials.append(cred)
+
+    selected = random.choice(host_credentials)
+    cleanup.remove(selected)
+    host_credentials.remove(selected)
+    selected.delete()
+
+    remote_host_credentials = HostCredential().list().json()
+    for local, remote in zip(host_credentials, remote_host_credentials):
+        local.equivalent(remote)
```

Comment:
This test is good but different from what the doc string describes because it does not "repeat" the deletion process utill are creds are gone. You could remove item `5)` from the doc string or update the test

Reply:
I stopped on step 4, will do as the docstring states.